### PR TITLE
UEFI installation fix

### PR DESCRIPTION
**Pull request by ArrayBolt3**
This PR contains a patch fixes the UEFI installation bug.

I followed the Lubuntu packaing tutorial, and I think I did everything right, but there's quite a bit of changes in this that I didn't expect (maybe leftovers from the debuild step?), so look this over carefully since I may have mangled something unintentionally.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#   SPDX-FileCopyrightText: no
```
Comment (tsimonq2) [inline .clang-format]: Delete this file

File: .gersemirc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#   SPDX-FileCopyrightText: no
```
Comment (tsimonq2) [inline .gersemirc]: Delete this file

File: .git-blame-ignore-revs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#   SPDX-FileCopyrightText: no
```
Comment (tsimonq2) [inline .git-blame-ignore-revs]: Delete file

File: .reuse/dep5 [hunk 4]
```diff
@@ -0,0 +1,90 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Calamares
```
Comment (tsimonq2) [inline .reuse/dep5]: Delete?

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,9 @@
+calamares (3.3.0~git20220610-0ubuntu2) kinetic; urgency=medium
+
+  * Fix UEFI installation bug https://github.com/calamares/calamares/issues/1990
```
Comment (tsimonq2) [inline debian/changelog]: Consider splitting this into multiple lines. It makes for a cleaner changelog.

File: debian/patches/uefi-bootloader-fix.patch [hunk 6]
```diff
@@ -0,0 +1,27 @@
+Description: Fix UEFI installation bug
+ A previous change in Calamares 3.3.0 caused the UEFI partition to not be
+ populated when installing Lubuntu on a UEFI system. This patch reverts the
+ change, allowing UEFI installations to work again.
+Author: Aaron Rainbolt <arraybolt3@gmail.com>
+Origin: upstream, https://github.com/calamares/calamares/blob/calamares/src/modules/bootloader/main.py
```
Comment (tsimonq2) [inline debian/patches/uefi-bootloader-fix.patch]: File path is unnecessary, upstream is sufficient

**Review by tsimonq2 — CHANGES_REQUESTED** [review]
Please make the changes requested. Besides that, LGTM.

**Comment by ArrayBolt3** [comment]
All requested changes made.

**Pull request merged**